### ISSUE TITLE
[BUG] advice +chinese--org-html-paragraph-a in modules/input/chinese/config.el doesn't not work

### ISSUE THREAD

**Issue by cnsunyour** [hlissner/doom-emacs#1649]
Advice +chinese--org-html-paragraph-a in modules/input/chinese/config.el does not work properly, and the number of parameters is incorrect when called. I tried to modify it to the following code and it runs successfully:
```
(defadvice! +chinese--org-html-paragraph-a (args)
  "Join consecutive Chinese lines into a single long line without
unwanted space when exporting org-mode to html."
  :filter-args #'org-html-paragraph
  (let* ((fix-regexp "[[:multibyte:]a-zA-Z0-9]")
         (paragraph (car args))
         (origin-contents (nth 1 args))
         (info (car (last args)))
         (fixed-contents
          (replace-regexp-in-string
           (concat "\\("
                   fix-regexp
                   "\\) *\\(<[Bb][Rr] */>\\)?\n *\\("
                   fix-regexp
                   "\\)")
           "\\1\\3"
           origin-contents)))
    (list paragraph fixed-contents info)))
```

**Comment by hlissner** [comment]
As of b2aec5f (on the straight branch, which will be merged into develop this weekend) this should be resolved for both the chinese and japanese modules. Appears to have simply been a careless mistake when the advise was converted from a `:before` to a `:filter-args` advice. Thanks for bringing it to my attention!

**Comment by cnsunyour** [comment]
Wow, this processing and reply is as fast as lightning! Thank you.
Um, I wonder if you noticed that I made some changes to the regular expression inside, because when I published the article using org2blog, I found that there would be some Chinese and English mixing and turning \n automatically into <br />\n Problem, only to make such changes to deal with these problems

**Comment by hlissner** [comment]
Ah, my bad. I did miss the regexp changes. I've merged them in as of ef4424b. Thanks for letting me know!

**Comment by cnsunyour** [comment]
It's my pleasure  to make some useful suggestions. ^_^
